package.json

```diff
@@ -28,7 +28,6 @@
     "@rei/cdr-col": "^0.1.0",
     "@rei/cdr-icon": "^0.2.0",
     "@rei/cdr-list": "^0.1.0",
-    "@rei/cdr-row": "^0.1.0",
-    "rei-cedar-tokens": "git+https://github.com/rei/rei-cedar-tokens.git#0.1.4"
+    "@rei/cdr-row": "^0.1.0"
```

Comment:
How are tokens pulled in now that this is gone?

Reply:
They come from `cdr-assets`. I was just cleaning up since this wasn't being used. 

I'm actually breaking tokens out into a separate package right now to make it easier to distribute them in different formats.